pkg/apis/networking/v1alpha1/clusteringress_defaults.go

```diff
@@ -55,4 +68,18 @@ func (p *HTTPClusterIngressPath) SetDefaults() {
 	if len(p.Splits) == 1 && p.Splits[0].Percent == 0 {
 		p.Splits[0].Percent = 100
 	}
+
+	if p.Timeout == nil {
+		p.Timeout = &metav1.Duration{Duration: DefaultTimeout}
+	}
+
+	if p.Retries == nil {
+		p.Retries = &HTTPRetry{
+			PerTryTimeout: &metav1.Duration{Duration: DefaultTimeout},
+			Attempts:      DefaultRetryCount,
+		}
+	}
+	if p.Retries.PerTryTimeout == nil {
+		p.Retries.PerTryTimeout = &metav1.Duration{Duration: DefaultTimeout}
+	}
```

Comment:
Do you want to split these (`./pkg/apis/networking/...`) changes into a separate PR?  They LGTM, thanks for the cleanup.

Reply:
Sure, will do